dor/service_layer/handlers/receive_package.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+from dor.domain.events import PackageSubmitted, PackageReceived
+from dor.service_layer.unit_of_work import UnitOfWork
+
+def receive_package(event: PackageSubmitted, uow: UnitOfWork, translocator: Any) -> None:
+    workspace = translocator.create_workspace_for_package(event.package_identifier)
+
+    received_event = PackageReceived(
+        package_identifier=event.package_identifier,
+        tracking_identifier='aintthatpeculiar',
```

Comment:
This identifier needs to be generated, maybe using `uuid`, maybe using something else or a component we create.